src/Symfony/Component/Yaml/Inline.php

```diff
@@ -717,9 +721,15 @@ private static function evaluateScalar($scalar, $flags, $references = [])
                         return null;
                     case 0 === strpos($scalar, '!php/const'):
                         if (self::$constantSupport) {
-                            $i = 0;
-                            if (\defined($const = self::parseScalar(substr($scalar, 11), 0, null, $i, false))) {
-                                return \constant($const);
+                            if (false === $const = substr($scalar, 11)) {
+                                if (\defined('')) {
+                                    return \constant('');
```

Comment:
do we actually want to support it, isnt `define('', ...)` nonsense anyway? Semantically i'd expect `!php/const` to require a value.

Reply:
I think I can get rid of this part. I thought it was supported but I think I was wrong.